src/website/views/components/button.pug

```diff
@@ -301,6 +301,7 @@ p.-text
 .example.-mb--3
   .-p--3
     each size in ['sm', 'md', 'lg', 'xl']
+      h6= '-'+ size
```

Comment:
@davidojedacl for being consistent we should maintain h6 or p tags for the sizes in examples and they should be inside the div (as the brand page)

Reply:
done